protocol/protobuf/proto/state_representation/space/joint/joint_state.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package state_representation.proto;
+
+import "state_representation/state.proto";
+
+message JointState {
+  State state = 1;
+  repeated string joint_names = 2;
+  repeated double positions = 3;
+  repeated double velocities = 4;
+  repeated double accelerations = 5;
+  repeated double torques = 6;
+}
+
+message JointPositions {
+  State state = 1;
+  repeated string joint_names = 2;
+  repeated double positions = 3;
+}
+
+message JointVelocities {
+  State state = 1;
+  repeated string joint_names = 2;
+  repeated double velocities = 4;
```

Comment:
same comment as @domire8 I guess this should be `3` here and torques in `JointTorques` as well